components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -3425,7 +3425,7 @@ def countAnnotations(self, obj_type, obj_ids=[]):
             return counts
 
         ctx = self.SERVICE_OPTS.copy()
-        ctx.setOmeroGroup(self.group)
+        ctx.setOmeroGroup(None)
```

Comment:
I found that when you are querying in a group that is NOT your default group, you actually need to use ```ctx.setOmeroGroup(-1)``` here, otherwise you don't see anything:

![screen shot 2017-02-28 at 14 56 40](https://cloud.githubusercontent.com/assets/900055/23410215/59045f1e-fdc6-11e6-8aa9-64dea028f82b.png)